src/views/protected/Recommendation.vue

```diff
@@ -1,32 +1,77 @@
 <template>
   <div class="box" style="text-align: center">
-    <div class="mb-5">
+    <div class="mb-5" @click="open_case">
       <Button></Button>
     </div>
-    <div class="space alert alert-secondary alert-dismissible shadow-inset fade show" role="alert">
+    <div
+      class="
+        space
+        alert alert-secondary alert-dismissible
+        shadow-inset
+        fade
+        show
+      "
+      role="alert"
+    >
       <h5>People you may know</h5>
       <button
         type="button"
         class="close top"
         data-dismiss="alert"
-        aria-label="Close">
+        aria-label="Close"
+      >
         <span aria-hidden="true">×</span>
       </button>
-      <div>
-        <PeopleCard></PeopleCard>
+      <div class="follow_cards">
+        <template
+          v-for="recommendation in recommendations"
+          :key="recommendation.recommends"
+        >
+          <PeopleCard :recommendation="recommendation" />
+        </template>
       </div>
     </div>
   </div>
 </template>
 <script>
 import Button from "@/components/recommendation/Button.vue";
 import PeopleCard from "@/components/recommendation/PeopleCard.vue";
+import { recommendationService } from "@/services";
+import { config } from "@/configurations";
+import { stringFormat } from "@/helpers";
 
 export default {
   name: "Recommendation",
-  components: {Button, PeopleCard},
+  components: { Button, PeopleCard },
+  data() {
+    return {
+      recommendations: null,
+    };
+  },
+  methods: {
+    getRecommendations() {
+      recommendationService
+        .getRecommendations()
+        .then((recommendations) => {
+          this.recommendations = recommendations;
+        })
+        .catch(() => {
+          throw stringFormat(
+            config.messagingConfig.messages.recommendation.failure,
+            "recommendations"
+          );
+        });
+    },
+    open_case() {
```

Comment:
@Chirag-Khatri-CK what is the purpose of this ?

Reply:
Sir actually I am rendering case.vue to check I make use of this.